### PR TITLE
docs: translate the content under the general directory

### LINKED ISSUE #1077: [Docs]: translate the content under the `general` directory
### Current State

**Description of the current state/issue**:
Translate the contents of the `general` directory, including: 
[join](https://github.com/apache/apisix-website/blob/master/website/docs/general/join.md)
[submit-issue](https://github.com/apache/apisix-website/blob/master/website/docs/general/submit-issue.md)
[security-guide](https://github.com/apache/apisix-website/blob/master/website/docs/general/security-guide.md)
[committer-guide](https://github.com/apache/apisix-website/blob/master/website/docs/general/committer-guide.md.) 

For documentation style and content specification, make sure to read [中文文档写作指南](https://apisix.apache.org/zh/docs/general/documentation-style-guide/) first before submitting content.

### PR DESCRIPTION
**Pull request by lunarwhite**
Fixes: #1077 

Changes: add zh-hans translations for following contents:

- [website/docs/general/join.md](https://github.com/apache/apisix-website/blob/master/website/docs/general/join.md)
- [website/docs/general/submit-issue.md](https://github.com/apache/apisix-website/blob/master/website/docs/general/submit-issue.md)
- [website/docs/general/security-guide.md](https://github.com/apache/apisix-website/blob/master/website/docs/general/security-guide.md)
- [website/docs/general/committer-guide.md](https://github.com/apache/apisix-website/blob/master/website/docs/general/committer-guide.md.)

### CODE REVIEW

File: website/i18n/zh/docusaurus-plugin-content-docs/current/committer-guide.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+如果你已经成为一个项目的有价值的贡献者，你可能会被邀请成为 committer。Committer 是 Apache 软件基金会（Apache Software Foundation，简称 ASF）的一个专用术语，象征着致力于某个特定项目的人。它为你带来了对项目仓库和资源的写入权限。
+
+更多的细节可以在[这里](https://community.apache.org/contributors/)找到。
+
+## 晋升
+
+Apache APISIX 社区遵循 Apache 社区接受新 committer 的流程。在贡献者积极参加 APISIX 社区的活动后，(P) PMC 成员将决定是否邀请该贡献者加入 Committers 和 (P) PMC。
+
+1. 此过程首先在 @private 中进行讨论和投票。只有当前的 PMC 成员可以提名一个新的 committer 或 PMC 成员。讨论中的每封新邮件都将延长该过程 72 小时。在 PMC 成员做决定时，请保持耐心。
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/committer-guide.md]:
```suggestion
1. 此过程首先在 @private 中进行讨论和投票。只有当前的 PMC 成员可以提名一个新的 committer 或 PMC 成员。讨论中的每封新邮件都将延长该过程 72 小时，所以请耐心等待结果。

```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/committer-guide.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+2. 如果投票通过，将向贡献者发送一封邀请信，邀请其成为 committer，并注明 @private CC'ed。
+
+3. 新的 committer 可以签署 [ICLA](https://www.apache.org/licenses/contributor-agreements.html#clas) 并申请 Apache ID 和电子邮件地址。
+
+4. 然后可以通过 [GitBox](https://gitbox.apache.org/setup/) 将 Apache 账户和 GitHub 账户关联起来。
+
+5. [团队](/team)页面将为新的 committer 更新。
+
+
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/committer-guide.md:37]:
```suggestion

```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/committer-guide.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+7. 改进流程和工具。
+
+8. 引导新的贡献者加入社区。
+
+## 如何成为 Apache APISIX PMC 成员？
+
+在 APISIX 社区，获得更多成绩的 committer 可以被邀请成为项目管理委员会（Project Management Committee，简称 PMC）的一员。
+
+当你初次接触 ASF 使用的开放开发流程的时，有一件事可能很难理解，那就是我们重视社区胜过重视代码。一个强大而健康的社区会受到尊重，并会成为一个既有意思又有价值的地方。更重要的是，一个多样化且健康的社区可以持续地支持代码的开发，即使个别公司没有持续在这里深耕。
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/committer-guide.md]:
```suggestion
Apache 社区的理念之一是“社区优于代码”。一个充满活力且持续成长的开源社区自然会受到开发者们的青睐，同时也会提供既有意思又有价值的内容。

```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/join.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+---
+id: join
+title: 参与社区对话
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/join.md]:
```suggestion
title: 参与社区讨论

```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/join.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+---
+id: join
+title: 参与社区对话
+keywords:
+  - Apache APISIX
+  - Slack
+  - APISIX 社区
+description: 加入 Apache APISIX 社区并参与对话。
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/join.md]:
```suggestion
description: 加入 Apache APISIX 社区并参与讨论。

```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/join.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+Apache APISIX 是一个由 Apache 软件基金会下的社区成员维护的项目，并遵守 [ASF 的行为准则](https://www.apache.org/foundation/policies/conduct.html)。
+
+我们有一个透明和包容的社区，你可以加入其中并为讨论做出贡献。
+
+## 订阅邮件列表
+
+你可以通过订阅 Apache APISIX 邮件列表来讨论问题，提出新的想法，并参与其他社区讨论。
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/join.md]:
```suggestion
你可以通过订阅 Apache APISIX 邮件列表来讨论问题或提出新的想法，并参与其他多种话题的社区讨论。

```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/security-guide.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+- API 网关
+- APISIX
+- Apache APISIX
+- 安全
+description: 如果你对 APISIX 的安全性有顾虑，或者发现了漏洞或潜在威胁，请不要犹豫，发送邮件到 security@apache.org 与 Apache 安全团队取得联系。
+---
+
+Apache 软件基金会在消除其软件项目中的安全问题方面采取了严格的立场。Apache APISIX 也非常关注与其特性和功能有关的安全问题。
+
+如果你对 APISIX 的安全性有顾虑，或者发现了漏洞或潜在威胁，请不要犹豫，发送邮件到 security@apache.org 与 Apache 安全团队取得联系。
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/security-guide.md]:
```suggestion
如果你对 APISIX 的安全性有顾虑，或者发现了漏洞或潜在威胁，请及时发送邮件到 security@apache.org 与 Apache 安全团队取得联系。

```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/security-guide.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+
+请在邮件中注明项目名称为 APISIX，产品名称为 APISIX 或 APISIX-Dashboard，并描述相关问题或潜在威胁。你需要推荐一种重现和复制该问题的方法。Apache 安全团队和 APISIX 社区将在评估并分析结果后向你反馈。
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/security-guide.md]:
```suggestion
请在邮件中注明项目名称为 `APISIX`，产品名称为 `APISIX` 或 `APISIX-Dashboard`，并描述相关问题或潜在威胁。你需要推荐一种如何复现该问题的方法。Apache 安全团队和 APISIX 社区将在评估并分析结果后向你反馈。

```

Comment (SylviaBABY) [reply website/i18n/zh/docusaurus-plugin-content-docs/current/security-guide.md]: 这里用代码格式仅代表强调目的^^

File: website/i18n/zh/docusaurus-plugin-content-docs/current/submit-issue.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+description: 关于提交 Bug 报告或功能请求的指南。
+---
+
+在提交 issue 时，请遵循此指南。
+
+在提交 issue 之前，尝试通过综合搜索来解决问题。
+
+[Issues](https://github.com/apache/apisix/issues) 和 [Discussions](https://github.com/apache/apisix/discussions) 是创建一个 issue 前搜索解决方案的好去处。
+
+要提交一个 issue：
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/submit-issue.md]:
```suggestion
如需提交 issue：

```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/submit-issue.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+1. 检查[已经创建的 issues](https://github.com/apache/apisix/issues) 以免重复。
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/submit-issue.md]:
```suggestion
1. 检查已经创建的[issues](https://github.com/apache/apisix/issues) 列表，避免重复操作。

```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/committer-guide.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
+---
+id: committer-guide
+title: 项目管理结构
+keywords:
+- API 网关
+- APISIX
+- Apache APISIX
+- APISIX committer 指南
+description: 这篇文章涵盖了 Apache APISIX committers 的一些准则，包括贡献的类型与贡献者的晋升路线，以及贡献者如何成为 committer，进一步成为 PMC 成员。
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/committer-guide.md]:
```suggestion
description: 本文介绍了 Apache APISIX committers 的一些准则，包括贡献的类型与贡献者的晋升路线，以及贡献者如何成为 committer，进一步成为 PMC 成员。

```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/submit-issue.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+
+2. 创建一个[新的 issue](https://github.com/apache/apisix/issues/new/choose) 并选择类型。
+
+3. 用一个明确的、描述性的标题来定义 issue。
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/submit-issue.md]:
```suggestion
3. 用一个明确且描述性强的标题来定义 issue，如 Bug、Doc 等关键词开头的标题。

```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/committer-guide.md [hunk 13]
```diff
@@ -0,0 +1,63 @@
+---
+id: committer-guide
+title: 项目管理结构
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/committer-guide.md]:
```suggestion
title: 项目管理体系

```

**Comment by SylviaBABY** [comment]
@lunarwhite Thanks for your contribution! And Great Work！

I have reviewed and left some comments, feel free to talk with me if you have any questions^ ^

File: website/i18n/zh/docusaurus-plugin-content-docs/current/committer-guide.md [hunk 14]
```diff
@@ -0,0 +1,63 @@
...
+
+1. 此过程首先在 @private 中进行讨论和投票。只有当前的 PMC 成员可以提名一个新的 committer 或 PMC 成员。讨论中的每封新邮件都将延长该过程 72 小时。在 PMC 成员做决定时，请保持耐心。
+
+2. 如果投票通过，将向贡献者发送一封邀请信，邀请其成为 committer，并注明 @private CC'ed。
+
+3. 新的 committer 可以签署 [ICLA](https://www.apache.org/licenses/contributor-agreements.html#clas) 并申请 Apache ID 和电子邮件地址。
+
+4. 然后可以通过 [GitBox](https://gitbox.apache.org/setup/) 将 Apache 账户和 GitHub 账户关联起来。
+
+5. [团队](/team)页面将为新的 committer 更新。
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/committer-guide.md]:
```suggestion
5. 新的 committer 需要更新[团队](/team)页面添加个人信息。

```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/committer-guide.md [hunk 15]
```diff
@@ -0,0 +1,63 @@
...
+
+2. 重构代码。
+
+3. 及时可靠地 Review PR。
+
+4. 考虑并接受功能请求。
+
+5. 回答问题。
+
+6. 更新文档和实例。
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/committer-guide.md]:
```suggestion
6. 更新文档和示例。
```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/join.md [hunk 16]
```diff
@@ -0,0 +1,35 @@
...
+
+1. 要订阅邮件列表，首先发送电子邮件到 dev-subscribe@apisix.apache.org。
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/join.md]:
```suggestion
1. 如果你需要订阅邮件列表，请发送电子邮件到 dev-subscribe@apisix.apache.org，邮件内容可为任意内容。

```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/join.md [hunk 17]
```diff
@@ -0,0 +1,35 @@
...
+
+2. 一旦你发送了邮件，你将收到一封来自 dev-help@apisix.apache.org 的确认邮件。
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/join.md]:
```suggestion
2. 一旦你发送了邮件，就会收到一封来自 dev-help@apisix.apache.org 的确认邮件。

```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/join.md [hunk 18]
```diff
@@ -0,0 +1,35 @@
...
+
+3. 回复此邮件以确认你的订阅。
+
+4. 一旦你确认了订阅，你将收到一封欢迎邮件，其中包含如何使用和通过邮件列表交流的说明。
+
+5. 要查看邮件列表的存档邮件，请查阅[历史](https://lists.apache.org/list.html?apisix.apache.org)。
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/join.md]:
```suggestion
5. 如果你想要查看邮件列表的存档邮件，请查阅[历史邮件](https://lists.apache.org/list.html?apisix.apache.org)。

```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/join.md [hunk 19]
```diff
@@ -0,0 +1,35 @@
...
+
+## 加入 Slack 频道
+
+你可以通过以下两种方式加入 Apache APISIX 的 Slack 频道：
+
+- 通过[此邀请链接](https://join.slack.com/t/the-asf/shared_invite/zt-vlfbf7ch-HkbNHiU_uDlcH_RvaHv9gQ)加入 Apache 软件基金会 Slack 工作区（_如果此链接已过期，请[创建一个 issue](./submit-issue.md)_），然后加入 **#apisix** 频道（频道 -> 浏览频道 -> 搜索 "apisix"）。
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/join.md]:
```suggestion
- 通过此[邀请链接](https://join.slack.com/t/the-asf/shared_invite/zt-vlfbf7ch-HkbNHiU_uDlcH_RvaHv9gQ)加入 Apache 软件基金会 Slack 工作区，然后加入 **#apisix** 频道（频道 -> 浏览频道 -> 搜索 "apisix"）。

:::tip

如果此邀请链接已过期，请[创建一个 issue](./submit-issue.md)）

:::
```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/security-guide.md [hunk 20]
```diff
@@ -0,0 +1,18 @@
+---
+id: security
+title: 报告安全漏洞
+keywords:
+- API 网关
+- APISIX
+- Apache APISIX
+- 安全
+description: 如果你对 APISIX 的安全性有顾虑，或者发现了漏洞或潜在威胁，请不要犹豫，发送邮件到 security@apache.org 与 Apache 安全团队取得联系。
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/security-guide.md]:
```suggestion
description: 如果你对 APISIX 的安全性有顾虑，或者发现了漏洞或潜在威胁，请及时发送邮件到 security@apache.org 与 Apache 安全团队取得联系。

```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/committer-guide.md (hunk 3 above)
Comment (SylviaBABY) [reply website/i18n/zh/docusaurus-plugin-content-docs/current/committer-guide.md]: 这里我觉得如果直翻会非常迷惑；不如直接用中文表达一下开源社区的一些特点或者独具的优势

File: website/i18n/zh/docusaurus-plugin-content-docs/current/security-guide.md [hunk 21]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+Apache 软件基金会在消除其软件项目中的安全问题方面采取了严格的立场。Apache APISIX 也非常关注与其特性和功能有关的安全问题。
+
+如果你对 APISIX 的安全性有顾虑，或者发现了漏洞或潜在威胁，请不要犹豫，发送邮件到 security@apache.org 与 Apache 安全团队取得联系。
+
+请在邮件中注明项目名称为 APISIX，产品名称为 APISIX 或 APISIX-Dashboard，并描述相关问题或潜在威胁。你需要推荐一种重现和复制该问题的方法。Apache 安全团队和 APISIX 社区将在评估并分析结果后向你反馈。
+
+请注意，在向公共区域披露安全问题之前，应先通过安全电子邮件报告安全问题。
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/security-guide.md]:
```suggestion
```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/security-guide.md [hunk 22]
```diff
@@ -0,0 +1,18 @@
...
+- APISIX
+- Apache APISIX
+- 安全
+description: 如果你对 APISIX 的安全性有顾虑，或者发现了漏洞或潜在威胁，请不要犹豫，发送邮件到 security@apache.org 与 Apache 安全团队取得联系。
+---
+
+Apache 软件基金会在消除其软件项目中的安全问题方面采取了严格的立场。Apache APISIX 也非常关注与其特性和功能有关的安全问题。
+
+如果你对 APISIX 的安全性有顾虑，或者发现了漏洞或潜在威胁，请不要犹豫，发送邮件到 security@apache.org 与 Apache 安全团队取得联系。
+
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/security-guide.md:15]: :::important

在向公共频道披露安全问题之前，请优先通过该邮件地址报告安全问题。

:::

File: website/i18n/zh/docusaurus-plugin-content-docs/current/submit-issue.md [hunk 23]
```diff
@@ -0,0 +1,28 @@
...
+- APISIX
+- Apache APISIX
+- Issue
+- Bug
+description: 关于提交 Bug 报告或功能请求的指南。
+---
+
+在提交 issue 时，请遵循此指南。
+
+在提交 issue 之前，尝试通过综合搜索来解决问题。
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/submit-issue.md]:
```suggestion
在提交 issue 之前，请优先通过综合搜索来解决问题。

```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/submit-issue.md [hunk 24]
```diff
@@ -0,0 +1,28 @@
...
+
+[Issues](https://github.com/apache/apisix/issues) 和 [Discussions](https://github.com/apache/apisix/discussions) 是创建一个 issue 前搜索解决方案的好去处。
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/submit-issue.md]:
```suggestion
在创建 issue 之前，你可以通过 [Issues](https://github.com/apache/apisix/issues) 和 [Discussions](https://github.com/apache/apisix/discussions) 搜索解决方案。

```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/submit-issue.md [hunk 25]
```diff
@@ -0,0 +1,28 @@
...
+
+1. 检查[已经创建的 issues](https://github.com/apache/apisix/issues) 以免重复。
+
+2. 创建一个[新的 issue](https://github.com/apache/apisix/issues/new/choose) 并选择类型。
+
+3. 用一个明确的、描述性的标题来定义 issue。
+
+4. 根据模板填写必要的信息。
+
+5. 参与该 issue 的讨论，并在必要时提供更多信息。
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/submit-issue.md]:
```suggestion
5. 请随时关注你的电子邮件，并参与该 issue 的讨论，在必要时提供更多信息。

```

**Review by guitu168 — COMMENTED** [review]
other LGTM.

File: website/i18n/zh/docusaurus-plugin-content-docs/current/submit-issue.md [hunk 26]
```diff
@@ -0,0 +1,28 @@
...
+- APISIX
+- Apache APISIX
+- Issue
+- Bug
+description: 关于提交 Bug 报告或功能请求的指南。
+---
+
+在提交 issue 时，请遵循此指南。
+
+在提交 issue 之前，请优先通过综合搜索来解决问题。
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/submit-issue.md]:
```suggestion
```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/join.md [hunk 27]
```diff
@@ -0,0 +1,41 @@
...
+
+1. 如果你需要订阅邮件列表，请发送电子邮件到 dev-subscribe@apisix.apache.org，邮件内容可为任意内容。
+
+2. 一旦你发送了邮件，就会收到一封来自 dev-help@apisix.apache.org 的确认邮件。
+
+3. 回复此邮件以确认你的订阅。
+
+4. 一旦你确认了订阅，你将收到一封欢迎邮件，其中包含如何使用和通过邮件列表交流的说明。
+
+5. 如果你想要查看邮件列表的存档邮件，请查阅[历史邮件]((https://lists.apache.org/list.html?apisix.apache.org)。
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/join.md]:
```suggestion
5. 如果你想要查看邮件列表的存档邮件，请查阅[历史邮件](https://lists.apache.org/list.html?apisix.apache.org)。

```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/join.md [hunk 28]
```diff
@@ -0,0 +1,41 @@
...
+
+## 加入 Slack 频道
+
+你可以通过以下两种方式加入 Apache APISIX 的 Slack 频道：
+
+- 通过此[邀请链接](https://join.slack.com/t/the-asf/shared_invite/zt-vlfbf7ch-HkbNHiU_uDlcH_RvaHv9gQ)加入 Apache 软件基金会 Slack 工作区，然后加入 **#apisix** 频道（频道 -> 浏览频道 -> 搜索 "apisix"）。
+
+:::tip
+
+如果此邀请链接已过期，请[创建一个 issue](./submit-issue.md)）
```
Comment (guitu168) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/join.md]:
```suggestion
如果此邀请链接已过期，请[创建一个 issue](./submit-issue.md)。

```

File: website/i18n/zh/docusaurus-plugin-content-docs/current/security-guide.md (hunk 22 above)
Comment (guitu168) [reply website/i18n/zh/docusaurus-plugin-content-docs/current/security-guide.md:15]: 请添加在这段话后面~

**Review by SylviaBABY — APPROVED** [review]
LGTM

**Review by guitu168 — APPROVED** [review]

**Review by guoqqqi — APPROVED** [review]

**Pull request merged**